### ISSUE TITLE
getting separate X, Y values

### ISSUE THREAD

**Issue by ludovico85** [jjimenezshaw/Leaflet.UTM#3]
Hi, is there a method for getting separate x and y values?
The function marker.getLatLng().utm() seems to retrieve only merged values.
thanks

**Comment by jjimenezshaw** [comment]
I don't know if I understand your question.
You can get just `x` doing `marker.getLatLng().utm().x`

**Comment by jjimenezshaw** [comment]
The method `utm()` returns an `object`, that can be converted easily to string. But you can always extract the separate values.
For a marker I run `marker.getLatLng().utm()` in the Console and got all the values:
```
L.Utm {x: 548308.5402722592,
       y: 4536684.767496558,
       zone: 30,
       band: "T",
       southHemi: false}
```

**Comment by jjimenezshaw** [comment]
I have added this documentation on README.md, and released 1.0.0 (no change in code)
